x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/agent_logs.tsx

```diff
@@ -9,6 +9,7 @@ import url from 'url';
 import { stringify } from 'querystring';
 
 import React, { memo, useMemo, useState, useCallback, useEffect } from 'react';
+import type { FunctionComponent } from 'react';
```

Comment:
any particular reason importing this vs just doing `React.FunctionComponent` like before?

Reply:
No other reason than my editor autocomplete I will change it back to the `React.FunctionComponent` way